src/service/resources-interface.js

```diff
@@ -123,6 +156,16 @@ export class ResourcesInterface extends MutatorInterface {
    */
   ampInitComplete() {}
 
+  /**
+   * @param {number} relayoutTop
+   */
+  setRelayoutTop(relayoutTop) {}
+
+  /**
+   * Flag that the height could have been changed.
+   */
+  maybeHeightChanged() {}
```

Comment:
I think this is a great change overall, but a side effect is that many private methods are now public.

Can we avoid this API pollution? E.g. move Resources/Mutator/Owners into their own subfolder `src/service/resources/` and mark previously-private methods with `@package`.

Reply:
That sounds great. That is an existing problem where we deal with today by using presubmit check to disallow any external dependency. The drawback is that js minimizer does not transform their method names, causing increased js bundle size.

Since this PR is already large. Can we merge this and then further refactor into a package?

Reply:
SGTM

> The drawback is that js minimizer does not transform their method names, causing increased js bundle size.

This is true for both `@public` and `@package` methods right?